package.json

```diff
@@ -11,8 +11,8 @@
     "dev": "lerna run dev --parallel",
     "dev:api": "yarn workspace @cdt/api dev",
     "dev:frontend": "yarn workspace @cdt/frontend dev",
-    "lint": "lerna run lint --parallel",
-    "start": "lerna run start --parallel",
+    "lint": "lerna run lint",
+    "start": "lerna run start",
```

Comment:
Can start really work without parallel ? Isn't it parallel blocking processes ?

Reply:
oups too late